lib/nerves_hub_link/client.ex

```diff
@@ -81,7 +81,9 @@ defmodule NervesHubLink.Client do
   """
   @spec update_available(update_data()) :: update_response()
   def update_available(data) do
-    case apply_wrap(mod(), :update_available, [data]) do
+    ret = apply_wrap(mod(), :update_available, [data])
+
+    case ret do
```

Comment:
```suggestion
    case apply_wrap(mod(), :update_available, [data]) do
```

Just because I know this was done while troubleshooting 🤣